packages/preview/valkyrie/0.2.0/CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+# [unreleased](https://github.com/jamesxX/valakyrie/releases/tags/)
```

Comment:
Shouldn't that say `0.2.0`?

Reply:
Also, the links in this file still point to the old repository.

Reply:
Resolved by removing changelog from this PR (you're right that this is unlikely to be the place they look).